tests/shared/common.go

```diff
@@ -144,6 +144,84 @@ func MakeModule(configVersion string) csmv1.Module {
 	return moduleObj
 }
 
+func MakeSecretPowerFlexWithZone(name, ns, _ string) *corev1.Secret {
```

Comment:
Please add function description above the method definition

Reply:
done.